datafn/python/tests/test_integration.py

```
@@ -0,0 +1,477 @@
+        t1 = await db.find_one("tasks", [{"field": "id", "operator": "eq", "value": "t1"}])
+        assert t1 is None
+
+
+# ---------------------------------------------------------------------------
+# 12. Sync end-to-end
+# ---------------------------------------------------------------------------
+class TestSync:
+    @pytest.mark.asyncio
+    async def test_push_and_pull(self):
+        """SYN-001/003: Push with mutations, pull back."""
+        server, db = _make_server()
```

Ruff (0.15.7)

[warning] Unpacked variable `db` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)